dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/MasterServer.java

```diff
@@ -128,7 +132,9 @@ public void initialized() {
 
         this.masterCoordinator.start();
 
+        this.clusterManager.registerWorkerGroupListener(this.workerGroupTaskDispatcherManager);
```

Comment:
Meaningless, if we choose to create WorkerGroupDispatcher if not exist in the manager.

Reply:
Change the logic to dynamically create the workerGroupTaskDispatcher when adding tasks.